core/src/main/java/org/frankframework/receivers/JavaListener.java

```diff
@@ -168,15 +167,10 @@ private Message processRequest(@Nonnull MessageWrapper<M> messageWrapper, @Nonnu
 			throw new ListenerException("JavaListener [" + getName() + "] is not opened");
 		}
 		log.debug("JavaListener [{}] processing correlationId [{}]" , getName(), messageWrapper.getCorrelationId());
-		Object object = parentSession.get(PipeLineSession.HTTP_REQUEST_KEY); //TODO dit moet weg
-		if (object != null) {
-			if (object instanceof HttpServletRequest request) {
-				ISecurityHandler securityHandler = new HttpSecurityHandler(request);
-				parentSession.setSecurityHandler(securityHandler);
-			} else {
-				log.warn("No securityHandler added for httpRequest [{}]", object::getClass);
-			}
-		}
+
+		ISecurityHandler securityHandler = new SpringSecurityHandler();
+		parentSession.setSecurityHandler(securityHandler);
```

Comment:
Heel raar dit, ik denk dat wanneer de security context bestaat deze overgenomen dient te worden, hij hoeft niet opnieuw gezet te worden. Dit is een oud overblijfsel van de SRP van NN, ik denk dat het helemaal er uit kan.